player.py

```diff
@@ -17,4 +17,24 @@ def main(self, display):
         pygame.draw.rect(display, (255, 0, 0), (self.x, self.y, self.width, self.height))
 
     def update_score(self, add_point=1):
-        self.score += add_point
\ No newline at end of file
+        self.score += add_point
+
+    #function to display the current score of the player on the game screen
+    #parameters:
+    #display: the pygame display object used by the game, it will have the score blit onto it
+    def display_score(self, display):
+        #Create a font object to write the score with
+        score_font = pygame.font.SysFont('cambria', 26)
```

Comment:
From reading the pygame doc, I think we should check that the font module was actually initiated properly before using it though idk where you would want to put the check.  This mostly wouldn't be a problem in modern systems though.